website/_includes/manual/transaction-control.md

```diff
@@ -4,7 +4,22 @@ By default, each individual test (FitNesse page) in flow mode is executed in a t
 
 If in flow mode, you can use the Commit and Rollback commands to control the transactions manually, but remember that a final rollback will be added at the end of the test. These commands have no additional arguments.
 
-In standalone mode, you will probably control transactions from outside DbFit. Utility commands to commit and rollback are still provided, if you need them, as part of the DatabaseEnvironment fixture. For example, use this table to rollback:
+In standalone mode, you will probably control transactions from outside DbFit. Utility commands to commit and rollback are still provided, if you need them. For example, use this table to rollback:
 
-    !|DatabaseEnvironment|
```

Comment:
If this is no longer supported then I'll have lots of ETL tests to update! Can we support both?